NumLib/DOF/MeshComponentMap.h

```diff
@@ -45,9 +45,24 @@ class MeshComponentMap final
     /// The order (BY_LOCATION/BY_COMPONENT) of components is the same as of the
     /// current map.
     ///
+    /// This is single-component version.
+    ///
     /// \param component_id  The global components id.
     /// \param components components that should remain in the created subset
-    MeshComponentMap getSubset(std::size_t const component_id,
+    MeshComponentMap getSubset(std::size_t const component_ids,
```

Comment:
misnomer: `component_ids` → `component_id`
Btw: Is that removed in a later commit in favour of the method below?